docs/overview/debugging/gdb_plugin.rst

```diff
@@ -0,0 +1,71 @@
+.. _tools:
+
+==========
+GDB Plugin
+==========
```

Comment:
```suggestion
.. _gdb_plugin:

===========
GDB Plug-In
===========
```

Reply:
Hi, @XuNeo !

I suggest "Plug-In" instead of "plugin" because it is the correct way of writing it in English.  You can see it [here](https://www.onelook.com/?w=plug-in), it shows up in all the main dictionaries.  If you change the search phrase to "plugin", you can see that the main dictionaries do not have word/phrase entries for this term.

The rest looks great!

Only one more thing in case you were not already aware:  with .MD files, the typical conversion to HTML causes the lines to wrap where you wrap the lines in the .MD file.  Thus:
```
This is a sentence
with multiple words.
```
will typically make it come out that way in the HTML.  However, with .RST files, you don't have to worry about that.  The above would turn out as:
```
This is a sentence with multiple words.
```
in the generated HTML.  This is something I am thankful for because I'm doing a lot of editing in the .RST files and it helps it to be more readable in a text editor with a narrow width.

It works both ways though, so no real need to change the text.  I just thought I would pass that on for the future, in case you were not already aware of it.

Kind regards,
Vic